### PR TITLE
Update migrate-email.md

### PR DESCRIPTION
**Pull request by msbemba**
Added people API to the steps

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6259

File: microsoft-365/business-video/moveto-microsoft-365/migrate-email.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ The migration process takes several steps and can take from several hours to a c
 1. Navigate to the [APIs page](https://console.developers.google.com/apis/library). 
 1. In the search bar, enter **Gmail API**.
 1. Select it and then choose **Enable**.
-1. Repeat this process for Google Calendar API and Contacts API. 
+1. Repeat this process for Google Calendar API, People API and Contacts API.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/moveto-microsoft-365/migrate-email.md]:
```suggestion
1. Repeat this process for Google Calendar API, People API, and Contacts API. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@twernermsft Please merge this PR. Thanks

**Pull request merged**